CMakeLists.txt

```diff
@@ -263,19 +276,14 @@ set(AGORA_SOURCES
   src/agora/dodecode.cc
   src/agora/radio_lib.cc
   src/agora/radio_calibrate.cc
-  src/mac/mac_thread_basestation.cc)
+  src/mac/mac_thread_basestation.cc
+  src/agora/txrx/packet_txrx.cc
+  src/agora/txrx/packet_txrx_radio.cc
+  src/agora/txrx/workers/txrx_worker.cc
+  src/agora/txrx/workers/txrx_worker_sim.cc
+  src/agora/txrx/workers/txrx_worker_argos.cc
```

Comment:
Maybe it is good opportunity to rename argos to hardwhare or sdr, since we file names have changed in this PR? Up to you to decide...

Reply:
I am going to leave it as argos, as we have a USRP worker.  When I move around the software frame / hardware framer in the next PR it might be a good time to remove the usrp completely and rename argos (maybe 1 will be hardware framer and 1 will be software framer.